app/scripts/modules/core/src/help/HelpMenu.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import { Dropdown, MenuItem, Glyphicon } from 'react-bootstrap';
+
+import './HelpMenu.less';
+
+export const HelpMenu = () => {
+  return (
+    <li className="help-menu">
+      <Dropdown id="help-menu-dropdown" pullRight={true}>
```

Comment:
Just curious, why do these dropdowns need `id`s? Is this a part of bootstrap?

Reply:
Yup, a warning gets printed to console that `id` is preferred for accessibility.  I figured it was better to comply with the warnings than add console spam for devs.